lib/utils/transpile-es6.js

```diff
@@ -22,6 +22,10 @@ module.exports = function(tree, description, options) {
   }
 
   var outputTree = map(tree, '**/*.js', function(content, relativePath) {
+    if (relativePath.match(/umd\.js$/)) {
+      return '';
+    }
```

Comment:
instead of `map` a `find` could be used: https://github.com/stefanpenner/broccoli-stew/blob/master/lib/find.js#L47 with an exclude matcher.